src/quetz_client/client.py

```diff
@@ -164,18 +164,25 @@ def yield_packages(
             yield Package(**user_json)
 
     def post_file_to_channel(self, channel: str, file: Path, force: bool = False):
-        url = f"{self.url}/api/channels/{channel}/upload/{file.name}"
-        body = open(file, "rb")
+        file_path = Path(file)
+        url = f"{self.url}/api/channels/{channel}/upload/{file_path.name}"
+        body = open(file_path, "rb")
+        body_bytes = body.read()
```

Comment:
Please use a context manager or the `pathlib.read_bytes` function.

https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_bytes